### ISSUE TITLE
Add winlog receiver

### ISSUE THREAD

**Issue by djaglowski** [open-telemetry/opentelemetry-collector-contrib#2333]
Dependent on #2268

**Comment by djaglowski** [comment]
@jmwilliams89 is working on this.

**Comment by rockb1017** [comment]
is it for windows event log?

**Comment by djaglowski** [comment]
Yes, and there is already an operator in otel-log-collection: https://github.com/open-telemetry/opentelemetry-log-collection/tree/main/operator/builtin/input/windows
